src/commands/util/Steam.ts

```diff
@@ -53,7 +55,7 @@ export class Steam extends Command
 
         try
         {
-            const username = int.options.getString("username");
+            const username = (int.options.get("username", true).value as string).trim();
```

Comment:
I'm being a hoe but `.trim()` is also unnecessary

Reply:
wrong, its a user inputted string so it has to be trimmed

Reply:
i m a idot